.github/workflows/package-manager-ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         # Maintenance and active LTS
         node-version: [14, 16]
```

Comment:
should we also add node 18?

Reply:
Will add it alongside node 20 in April, just so i can combine two changes into one, unless someone else wants to add it prior to that. :)